src/wawa-table.ts

```diff
@@ -213,16 +213,16 @@ export class WawaTable extends LitElement {
     }
 
     protected render(): TemplateResult {
-        return html`${this.renderStyles()}<div style="width:100%;" @scroll=${this.onScroll}>
+        return html`${this.renderStyles()}<div class="table-container" style="width:100%;" @scroll=${this.onScroll}>
```

Comment:
This allows a subclass to wrap the component in another div and not break query selectors